include/SDL_video.h

```diff
@@ -278,6 +278,26 @@ typedef enum
     SDL_GL_CONTEXT_RESET_LOSE_CONTEXT    = 0x0001
 } SDL_GLContextResetNotification;
 
+typedef void *SDL_GLImageKHR;
+
+struct SDL_Dmabuf {
+    int fd;
+    uint32_t offset;
+    uint32_t stride;
+    size_t size;
+    void *virt;
```

Comment:
We probably need DRM modifiers here too.

Reply:
yes, I'll make a change